cmd/policy/pull.go

```diff
@@ -4,13 +4,15 @@ import "github.com/pkg/errors"
 
 type PullCmd struct {
 	Policies []string `arg:"" name:"policy" help:"Policies to pull from the remote registry."`
+	Untar    bool     `name:"untar" help:"Untar the policy bundle after pulling."`
```

Comment:
Why have two flags? It feels redundant, as you always have to validate `UntarDir`, so you might as well use the fact that when `UntarDir` is set instead of `--untar`